brightray/browser/browser_main_parts.cc

```diff
@@ -55,6 +60,8 @@
 
 namespace brightray {
 
+void OverrideMacAppLogsPath();
```

Comment:
The `OverrideMacAppLogsPath` is probably better to be a member of `BrowserMainParts ` to make things clearer, you can reference how `InitializeMainNib` was done.